contracts/script/deploy/DeployNitro.s.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.12;
+
+import "forge-std/Script.sol";
+import "src/interfaces/IEigenDAStructs.sol";
+import "src/core/EigenDAThresholdRegistry.sol";
+import "src/core/EigenDACertVerifier.sol";
+import "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
+import "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+
+import "forge-std/console2.sol";
+
```

Comment:
add comment stating this only ever ran once and will be immediately deprecated wen v2 hit mainnet